fdm-app/app/components/ui/textarea.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+
+import { cn } from "~/lib/utils"
+
+const Textarea = React.forwardRef<
+  HTMLTextAreaElement,
+  React.ComponentProps<"textarea">
+>(({ className, ...props }, ref) => {
+  return (
```

Comment:
please check the >(({ here.

Reply:
I checked it and looks valid. It closes the `forwardRef` of line 5